content/tutorial/tutorial.md

```diff
@@ -12,185 +12,190 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+यह ट्यूटोरियल किसी भी मौजूदा React ज्ञान को ग्रहण नहीं करेगा।
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## ट्यूटोरियल शुरू करने से पहले {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+हम इस ट्यूटोरियल के दौरान एक छोटा Game बनाएंगे। **आप इसे छोड़ सकते हैं क्योंकि आप Game नहीं बना रहे हैं -- लेकिन इसे एक मौका दें।** ट्यूटोरियल में आपके द्वारा सीखी जाने वाली तकनीकें किसी भी React ऐप्स के निर्माण के लिए मौलिक हैं, और इसमें महारत हासिल करने से आपको React की गहरी समझ मिलेगी।
 
->Tip
+>सलाह
 >
->This tutorial is designed for people who prefer to **learn by doing**. If you prefer learning concepts from the ground up, check out our [step-by-step guide](/docs/hello-world.html). You might find this tutorial and the guide complementary to each other.
+>यह ट्यूटोरियल उन लोगों के लिए बनाया गया है, जो **करके सीखना** पसंद करते हैं। यदि आप नीचे से ऊपर अवधारणाओं को सीखना पसंद करते हैं, हमारे [क्रमशः मार्गदर्शिका](/docs/hello-world.html) देखें। आपको यह ट्यूटोरियल और गाइडमार्गदर्शिका एक-दूसरे के संपूरक लग सकते हैं।
 
-The tutorial is divided into several sections:
+ट्यूटोरियल को कई खंडों में विभाजित किया गया है:
 
-* [Setup for the Tutorial](#setup-for-the-tutorial) will give you **a starting point** to follow the tutorial.
-* [Overview](#overview) will teach you **the fundamentals** of React: components, props, and state.
-* [Completing the Game](#completing-the-game) will teach you **the most common techniques** in React development.
-* [Adding Time Travel](#adding-time-travel) will give you **a deeper insight** into the unique strengths of React.
+* [ट्यूटोरियल के लिए व्यवस्था](#setup-for-the-tutorial) आपको ट्यूटोरियल का पालन करने के लिए **एक प्रारंभिक बिंदु** देगा।
+* [सिंहावलोकन](#overview) आपको सिखाएगा React की **बुनियादी बातें**: कौम्पोनॅन्ट्स, props और स्टेट।
```

Comment:
अवलोकन is a better translation

Reply:
also, should be reworded to आपको React की **बुनियादी बातें**: कौम्पोनॅन्ट्स, props और स्टेट सिखाएगा